convert_object_to_sorted_array.gs

```diff
@@ -0,0 +1,35 @@
+/**
+* Convert JSON object to array sorted in descending order of their values
+* @param {Object} object will have key value pair
+* @param {Object} processIdWithGCPId will have key value pair with key as processId and value as Cloud ProjectId
+* @return {Object} Sorted Array of the JSON object in descending order
+*/
+function convertObjectToSortedArray(object, processIdWithGCPId) {
+  var array = new Array();
+  for(var i in object) {
+    array.push({"key" : i, "value" : object[i], "GCPId" : processIdWithGCPId[i]});
+  }
+  
+  // get the sorted array
+  var sortedArray = sortByKey(array,"value");
+  return sortedArray;
+}
+
+
+/**
+* Convert JSON object to array sorted in descending order of their values
+* @param {Object} object will have key value pair
+* @param {String} projectId of the cloud project
+* @return {Object} Sorted Array of the JSON object in descending order
+*/
+function convertObjectToSortedArrayForSpecificProject(object, projectId) {
```

Comment:
I didn't get what this function is doing?

Reply:
I removed it as it wasn't needed now